### DISCUSSION TITLE
Synchronize the maps with the account

### DISCUSSION THREAD

**Discussion by chencl0214** [ppy/osu#22540] [Ideas] [answered]
Since I now have more and more maps (2000+), I want a function that associates my current maps record with my account, so that I can download these previously owned maps in the game when I log in to the game account after changing the computer,and in order to choose those to download, or which to delete.It may not be easy to express. In short, it is to bind the map record of the current computer with the account, so that these maps can be downloaded directly after changing the computer.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/discussions/17492
https://github.com/ppy/osu/issues/2351
